spec/models/authenticator/amazon_spec.rb

```diff
@@ -1,79 +1,48 @@
-require "aws-sdk-v1"
+require 'aws-sdk'
 
 describe Authenticator::Amazon do
+  AWS_ROOT_USER_KEY = 'aws_root_key'
```

Comment:
both decide on the role,access level,privilege of the user that connects to aws. 
So they are used as a string identifier throughout this spec. To me, this is a hiding magic numbers/strings in constants. 
Does that make sense?

````
        case access_key_id
          when AWS_ROOT_USER_KEY
            stub_iam_responses_for_root(resource)
          when AWS_IAM_USER_KEY
            get_user_response = resource.client.stub_data(:get_user).to_h
            resource.client.stub_responses(:get_user, get_user_response)
            resource.client.stub_responses(:list_users, 'AccessDenied')
        end
````